instance/models/server.py

```diff
@@ -316,18 +319,23 @@ def update_status(self):
 
```

Comment:
This TODO can be removed, because you're now checking for `Status.Terminated` on line 322.